packages/react-notifications/src/components/notification-kinds/unexpected-error/unexpected-error.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  NOTIFICATION_DOMAINS,
+  NOTIFICATION_KINDS_PAGE,
+  TAppNotification,
+  TAppNotificationValuesUnexpectedError,
+} from '@commercetools-frontend/constants';
+import { Spacings } from '@commercetools-frontend/ui-kit';
+import Notification from '../../notification';
+import apiErrorMessages from '../api-error-message/messages';
+
+type Props = {
+  notification: TAppNotification<{
+    domain: typeof NOTIFICATION_DOMAINS.PAGE;
+    kind: typeof NOTIFICATION_KINDS_PAGE['unexpected-error'];
+    values: TAppNotificationValuesUnexpectedError;
```

Comment:
FYI: you can see here how the type system helps avoiding mistakes

<img width="691" alt="Screenshot 2019-07-25 at 14 13 17" src="https://user-images.githubusercontent.com/1110551/61934991-d0cd3300-af89-11e9-8b0e-04ae20225825.png">
<img width="570" alt="Screenshot 2019-07-25 at 14 13 26" src="https://user-images.githubusercontent.com/1110551/61934992-d0cd3300-af89-11e9-88d7-a1ad593c03f5.png">